--- debian/changelog.orig	2016-09-19 23:37:25.157879132 +0000
+++ debian/changelog	2016-09-19 23:37:32.141853134 +0000
@@ -1,3 +1,15 @@
+linux (3.16.36-1+deb8u2) acs; urgency=high
+
+  * add driver patches for MLNX SN2700
+
+ -- Guohan Lu <gulv@microsoft.com>  Sun, 19 Dec 2015 01:50:04 +0100
+
+linux (3.16.36-1+deb8u2) acs; urgency=high
+
+  * add support for S6000
+
+ -- Shuotian Cheng <shuche@microsoft.com>  Sun, 19 Dec 2015 01:50:04 +0100
+
 linux (3.16.36-1+deb8u2) jessie-security; urgency=high

  * KEYS: Fix short sprintf buffer in /proc/keys show function (CVE-2016-7042)
